### PR TITLE
Added a deployment.yaml for kubernetes

### PR DESCRIPTION
**Pull request by jjasghar**
- Added a deployment yaml for k8s
- added steps to do it in the readme

Signed-off-by: JJ Asghar <jjasghar@gmail.com>

### CODE REVIEW

File: deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      app: discord-karma
+  template:
+    metadata:
+      labels:
+        app: discord-karma
+    spec:
+      containers:
+      - name: discord-karam
+        # a public image on quay
+        image: quay.io/jjasghar/discord-karma:1.0.0
```
Comment (Puhapig) [inline deployment.yaml:18]: I'd prefer not to have this direct reference in the file. Can you use a template instead?

**Review by Puhapig — CHANGES_REQUESTED** [review]
Remove specific reference to hosted registry.

File: deployment.yaml (hunk 1 above)
Comment (jjasghar) [reply deployment.yaml:18]: By template you mean something like "PUT IMAGE HERE" or something?

Comment (Puhapig) [reply deployment.yaml:18]: I'd say default to a local image name e.g. `discord-karma:1.0.0` and a describing that this should be overridden when it's deployed.

Comment (jjasghar) [reply deployment.yaml]: updated.

**Review by Puhapig — APPROVED** [review]

**Pull request merged**